File: solutions/2580-circular-sentence/solution.py
class Solution(object):
    def isCircularSentence(self, sentence):
        """
        :type sentence: str
        :rtype: bool
        """
        n=sentence.split()
        n1=len(n)
        if n1==1:
            return n[0][0]==n[0][-1]
        if sentence[0]==sentence[-1]:
            for i in range(n1-1):
                if not n[i][-1]==n[i+1][0]:
                    return False
        else:
            return False
        return True

File: solutions/2580-circular-sentence/solution.java
class Solution {
    public boolean isCircularSentence(String sentence) {
        String[] ls=sentence.split("\\ ");
        int n=ls.length;
        for(int i=1;i<n;i++)
        {
            if(ls[i].charAt(0)!=ls[i-1].charAt(ls[i-1].length()-1))
                return false;
        }
        return ls[0].charAt(0)==ls[n-1].charAt(ls[n-1].length()-1);
    }
}
